containers/README.md

```diff
@@ -18,13 +18,17 @@ To run containerized Kraken as a Kubernetes/OpenShift Deployment, follow these s
 3. Switch to `<namespace>` namespace:
     - In Kubernetes, use `kubectl config set-context --current --namespace=<namespace>`
     - In OpenShift, use `oc project <namespace>`
-4. Create a ConfigMap named kube-config using `kubectl create configmap kube-config --from-file=<path_to_kubeconfig>`
-5. Create a ConfigMap named kraken-config using `kubectl create configmap kraken-config --from-file=<path_to_kraken_config>`
-6. Create a ConfigMap named scenarios-config using `kubectl create configmap scenarios-config --from-file=<path_to_scenarios_folder>`
-7. Create a ConfigMap named scenarios-openshift-config using `kubectl create configmap scenarios-openshift-config --from-file=<path_to_scenarios_openshift_folder>`
-8. Create a ConfigMap named scenarios-kube-config using `kubectl create configmap scenarios-kube-config --from-file=<path_to_scenarios_kube_folder>`
-9. Create a service account to run the kraken pod `kubectl create serviceaccount useroot`.
-10. In Openshift, add privileges to service account and execute `oc adm policy add-scc-to-user privileged -z useroot`.
-11. Create a Job using `kubectl apply -f kraken.yml` and monitor the status using `oc get jobs` and `oc get pods`.
-
-NOTE: It is not recommended to run Kraken internal to the cluster as the pod which is running Kraken might get disrupted.
+4. Create a ConfigMap named kube-config using `kubectl create configmap kube-config --from-file=<path_to_kubeconfig>` *(eg. ~/.kube/config)*\
+***NOTE:** your kubeconfig might contain several cluster contexts and credentials so be sure, before creating the ConfigMap, to keep **only** the credentials related to the destination cluster. \
```

Comment:
Typo: `***NOTE:**` instead of `***NOTE***` ; it leaves `***` unterminated

Reply:
This is throughout the whole file not just this one reference

Reply:
Considering that we have a "global" Note section at the end of the Readme what do you think if I do something like this: 
`
***NOTES:** 
- It is not generally recommended to run Kraken internal to the cluster as the pod which is running Kraken might get disrupted, the suggested use case to run kraken from inside k8s/OpenShift is to target **another** cluster (eg. to bypass network restrictions or to leverage cluster's computational resources)*
- your kubeconfig might contain several cluster contexts and credentials so be sure, before creating the ConfigMap, to keep **only** the credentials related to the destination cluster. Please refer to the [Kubernetes documentation](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) for more details*
`

Reply:
I think those two notes are good to maybe have at the top before starting the steps, but the note about the scenarios would probably be better kept in the steps it goes with; thoughts?

Reply:
:+1: